File: backend/src/main/java/ca/metricalsky/winston/repository/fetch/FetchOperationRepository.java
package ca.metricalsky.winston.repository.fetch;

import ca.metricalsky.winston.entity.fetch.FetchOperationEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface FetchOperationRepository extends JpaRepository<FetchOperationEntity, Long> {

}

File: database/src/main/kotlin/ca/metricalsky/winston/database/repository/fetch/FetchOperationRepository.kt
package ca.metricalsky.winston.database.repository.fetch

import ca.metricalsky.winston.database.entity.fetch.FetchOperationEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface FetchOperationRepository : JpaRepository<FetchOperationEntity, Long>
